itests/worker_test.go

```diff
@@ -730,3 +730,83 @@ waitForProof:
 	require.NoError(t, params.UnmarshalCBOR(bytes.NewBuffer(slmsg.Params)))
 	require.Equal(t, abi.RegisteredPoStProof_StackedDrgWindow2KiBV1_1, params.Proofs[0].PoStProof)
 }
+
+func TestWorkerPledgeExpireCommit(t *testing.T) {
+	kit.QuietMiningLogs()
+	_ = logging.SetLogLevel("sectors", "debug")
+
+	var tasksNoC2 = kit.WithTaskTypes([]sealtasks.TaskType{sealtasks.TTAddPiece, sealtasks.TTDataCid, sealtasks.TTPreCommit1, sealtasks.TTPreCommit2, sealtasks.TTCommit2,
+		sealtasks.TTUnseal, sealtasks.TTFetch, sealtasks.TTCommit1, sealtasks.TTFinalize, sealtasks.TTFinalizeUnsealed})
+
+	fc := config.DefaultStorageMiner().Fees
+	fc.MaxCommitGasFee = types.FIL(abi.NewTokenAmount(10000)) // 10000 attofil, way too low for anything to land
+
+	ctx := context.Background()
+	client, miner, worker, ens := kit.EnsembleWorker(t, kit.WithAllSubsystems(), kit.ThroughRPC(), kit.WithNoLocalSealing(true),
+		kit.MutateSealingConfig(func(sc *config.SealingConfig) {
+			sc.BatchPreCommits = false
+			sc.AggregateCommits = true
```

Comment:
I do not understand why you are using aggregate commits to test cron activation failing.  Aggregate commits should not be exercising the cron activation code at all.

Reply:
I don't aggregate commits, I just use the commit-aggregation logic to "hold" sectors in the wait-for-aggregation state - until the start epoch passes - but when I submit the messages, they are submitted individually because the basefee is low